kubernetes-itests/src/test/java/io/fabric8/commons/ClusterEntity.java

```diff
@@ -49,6 +50,17 @@ public static void remove(InputStream inputStream) {
     }
   }
 
+  public static boolean kubernetesVersionAtLeast(String majorVersion, String minorVersion) {
```

Comment:
One last comment:
I'm not sure this method behaves as expected.
I suspect the output of `kubernetesVersionAtLeast("1", "19")` is false when checking a `2.0.0`/`2.0` cluster, which is not the expected output.